### PR TITLE
[SDAO-41] Edit manual, add new CLIs descriptions.

### PR DESCRIPTION
**Pull request by awson**
## Description
I've extended the manual with new CLIs descriptions.

Also I've edited some existing wordings

## Related issue(s)
[SDAO-41](https://issues.serokell.io/issue/SDAO-41)

### CODE REVIEW

File: MANUAL.md [hunk 1]
```diff
@@ -313,6 +343,98 @@ For example (for 2 keys of old Operations team and 3 keys of new Operations team
...
+stkr-token-cli set-successor ${env} --stkrAlias stkr --succAlias succ --msigAlias msig --msigKeyAlias stkrOps1 --print-sigs
+```
+
+Output from each Operations team member should be collected
+and then used in a command to set the successor of frozen STKR contract
+on the network (signatures shall be supplied just like in the above example):
+
+```
+stkr-token-cli set-successor ${env} --stkrAlias stkr --succAlias succ --msigAlias msig --msigSig <..sig1..> --msigSig <..sig2..>
+```
```
Comment (gpevnev) [inline MANUAL.md]: is whole upgradability thing tested on babylonnet?

Comment (georgeee) [reply MANUAL.md]: tested partially

Comment (georgeee) [reply MANUAL.md]: commands tested, not the whole flow

Comment (georgeee) [reply MANUAL.md]: would be good to test the whole flow, but I'll leave it for future

**Pull request merged**